platform/jobs/edxPlatformAccessibilityPr.groovy

```diff
@@ -57,18 +57,8 @@ Map privateJobConfig = [
     workerLabel: 'js-worker',
     whitelistBranchRegex: /^((?!open-release\/).)*$/,
     context: 'jenkins/a11y',
-    triggerPhrase: /.*jenkins\W+run\W+a11y.*/
-]
-
-Map python3JobConfig = [
-    open : true,
-    jobName : 'edx-platform-python3-accessibility-pr',
-    repoName : 'edx-platform',
-    workerLabel: 'js-worker',
-    whitelistBranchRegex: /^((?!open-release\/).)*$/,
-    context: 'jenkins/python3.5/a11y',
-    triggerPhrase: /.*jenkins\W+run\W+py35-django111\W+a11y.*/,
-    toxEnv: 'py35-django111'
+    triggerPhrase: /.*jenkins\W+run\W+a11y.*/,
+    pythonVersion: '3.5',
```

Comment:
^ if we are including `pythonVersion` in the private job, we need to have it in the public job too, right?

Reply:
Yes, apparently missed that one when making the updates.  Fixing.